common/static/common/js/spec/components/paging_collection.js

```diff
@@ -32,6 +32,44 @@ define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone)
             this.currentPage = currentPage;
             this.start = start;
             return response.items;
+        },
+
+        setPage: function (page) {
```

Comment:
On further reading, I see that I misread this. This is a mock collection used just for testing? Is this to avoid having server interactions? I'd rather use the true collection and mock the AJAX calls.